pkg/cli/requestproject/request_project.go

```diff
@@ -57,7 +57,7 @@ const (
 	requestProjectNewAppOutput = `
 You can add applications to this project with the 'new-app' command. For example, try:
 
-    %[1]s new-app django-psql-example
+    %[1]s new-app ruby~https://github.com/sclorg/ruby-ex.git
```

Comment:
@gabemontero does the rails ephemeral template still work? `rails-postgresql-example`

Reply:
it wasn't in the "fips approved" list @adambkaplan and ATM I have not idea how to set up a fips env to try it out.

I'll ask in the bug.

Reply:
So it is 10 months later, and with https://bugzilla.redhat.com/show_bug.cgi?id=1866597
turns out we should have gone with your suggestion @adambkaplan :-)

Revisiting the BZ for this PR, I never did get an answer to my question in https://bugzilla.redhat.com/show_bug.cgi?id=1774318#c2 on fips approval for ` rails-postgresql-example`

Upon review, ruby/rails components are the same for both our official template and the upstream repo wrt FIPS.